OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -53,23 +55,51 @@ public class AircraftInfo : ITraitInfo, IFacingInfo, IOccupySpaceInfo, ICruiseAl
 		[Desc("The upgrades to grant to self while airborne.")]
 		public readonly string[] AirborneUpgrades = { };
 
+		[Desc("Can the actor hover in place mid-air? If not, then the actor will have to remain in motion (circle around).")]
+		public readonly bool CanHover = false;
+
+		[Desc("Will this actor try to land after it has no more commands?")]
+		public readonly bool LandWhenIdle = true;
+
+		[Desc("Does this actor need to turn before landing?")]
+		public readonly bool TurnToLand = false;
+
+		public readonly WDist LandAltitude = WDist.Zero;
+
+		[Desc("How fast this actor ascends or descends when using horizontal take off/landing.")]
+		public readonly WAngle MaximumPitch = WAngle.FromDegrees(10);
+
+		[Desc("How fast this actor ascends or descends when using vertical take off/landing.")]
+		public readonly WDist AltitudeVelocity = new WDist(43);
```

Comment:
This can probably be left for a future PR, since it is moved from existing code.